backend/service/aws/dynamodb.go

```diff
@@ -6,18 +6,23 @@ import (
 	"github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
 	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 
 	dynamodbv1 "github.com/lyft/clutch/backend/api/aws/dynamodb/v1"
 )
 
+const maxDynamodbReadCapacityUnits = 35000
```

Comment:
Where these values are coming from?

Reply:
Good call out — they're set as constants in the jenkins' job that I'm recreating, but I should probably pull them out into config variables instead so users can set their own limits per their needs